index.html

```diff
@@ -0,0 +1,95 @@
+      </div>
+    </section>
+    <section>
+      <div id="section3">
+        <h2 class="text-center">section Three</h2>
+        <p><span>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Velit excepturi doloribus ratione nemo in, odio ab eos voluptatum earum voluptatem id iure alias voluptates saepe iste tempore expedita esse quod?</span><span>Atque, tenetur? Laborum minima cumque, velit reprehenderit veniam harum, expedita deserunt ratione, rem enim eligendi. Facere omnis sequi aperiam non numquam repudiandae officiis tempore pariatur a voluptatem, eveniet ea. Ipsam?</span><span>Laboriosam sed ipsum cumque facilis, saepe delectus ea nulla assumenda itaque minus alias odit sequi doloribus consectetur vero, excepturi odio harum. Illum a ratione doloribus laboriosam fugiat minima dolore voluptatem!</span><span>Ipsam excepturi animi saepe molestiae dolor voluptatem numquam alias sed nesciunt impedit, neque facere voluptates iure fugit quidem, omnis possimus. Itaque quas ea dolor facere corrupti, in omnis excepturi accusamus?</span><span>Atque distinctio perspiciatis ducimus ex officia esse animi, quas nesciunt mollitia velit magnam dolores veritatis doloremque non ipsam consequuntur? Veritatis consequuntur quia quae dolorum optio error reiciendis modi consequatur accusantium.</span><span>Corrupti dicta libero, optio eaque magnam doloribus perferendis illo, explicabo tempore laboriosam deleniti qui. Sequi ratione eligendi architecto quod repudiandae. Perferendis, commodi? Accusantium, quasi qui! Et veniam corporis sint sit.</span><span>Ex velit quae culpa? Accusamus at aut ut alias nihil odio consequuntur, cum quibusdam in nostrum similique eligendi harum voluptatum? Veniam blanditiis adipisci placeat dolore velit harum tempora amet ut.</span><span>Quia nisi iste porro consequatur ipsum molestias fuga qui accusantium beatae voluptatem animi sit repudiandae accusamus molestiae laborum, dignissimos labore nesciunt delectus dolores eos facere corporis rerum adipisci repellat. Distinctio!</span><span>Dolores facere voluptate tenetur laudantium in voluptates dolor ducimus! Maiores facilis nesciunt minima perspiciatis laborum iste eveniet, quisquam iusto consequuntur ad quam sapiente! Provident reprehenderit optio eaque molestiae fuga nemo.</span><span>Repellendus dolorum itaque earum, debitis obcaecati dolores praesentium id libero aut quam enim sequi laboriosam expedita. Vero repellendus velit illo omnis debitis quas? Hic suscipit ipsa debitis, deleniti ipsum illum!</span></p>
+      </div>
+    </section>
+
+  </div>
```

Comment:
- [ ] It would be better if an empty space is not left in your code. Leaving empty spaces does not make your code to follow HTML && CSS best practices